### PR TITLE
Update Masonry to 1.1.0

### PR DESCRIPTION
**Pull request by SergeyKuryanov**
Masonry updated to (currently) latest 1.1.0 version.

Example app updated and runs with no layout issues.

Tests not affected.

### CODE REVIEW

**Comment by SergeyKuryanov** [comment]
@adamyanalunas seems like we need to bump minimum version to iOS 8. Not sure that I allowed to do this.

**Comment by adamyanalunas** [comment]
@SergeyKuryanov To drop iOS 7 I had to do a lot of work that conflicts with your changes in #20. Merge in `master` to your branch and it should simplify this PR.

**Comment by SergeyKuryanov** [comment]
@adamyanalunas seems I merge your changes, but have some changes in `Gemfile.lock` file. Is this expected?

**Comment by adamyanalunas** [comment]
`master` uses the latest CocoaPods gem, 1.3.1. Run `bundle install` and your Gemfile.lock should match what's in HEAD.

**Comment by SergeyKuryanov** [comment]
@adamyanalunas thank you for tip, `bundle update` helped.

File: Example/Podfile.lock [hunk 1]
```diff
@@ -14,15 +14,15 @@ DEPENDENCIES:
...
+  Expecta: e1c022fcd33910b6be89c291d2775b3fe27a89fe
+  loudspeaker: 55f868639bb17d57c38fea1752848ef20e762e88
+  Masonry: 678fab65091a9290e40e2832a55e7ab731aad201
+  OCMock: f3f61e6eaa16038c30caa5798c5e49d3307b6f22
+  Specta: ac94d110b865115fe60ff2c6d7281053c6f8e8a2
 
 PODFILE CHECKSUM: f09561418a350e655ac9af3b4c31e8a50323005f
 
-COCOAPODS: 1.3.1
+COCOAPODS: 1.1.1
```
Comment (adamyanalunas) [inline Example/Podfile.lock]: This lockfile was still generated with an older version of CocoaPods. Try `bundle exec pod install` to fix.

File: Example/Podfile.lock [hunk 2]
```diff
@@ -1,10 +1,10 @@
 PODS:
-  - Expecta (1.0.6)
+  - Expecta (1.0.5)
```
Comment (adamyanalunas) [inline Example/Podfile.lock]: Why downgrade Expecta and other testing libraries?

**Review by adamyanalunas — CHANGES_REQUESTED** [review]

File: Example/Podfile.lock (hunk 2 above)
Comment (SergeyKuryanov) [reply Example/Podfile.lock]: Didn't downgrade by purpose. Will investigate.

**Comment by SergeyKuryanov** [comment]
@adamyanalunas now seems like everything is updated properly.

**Review by adamyanalunas — APPROVED** [review]
😎

**Pull request merged**